cuappdev/pear-ios
# Consolidate login, onboarding, and match screens

## Overview


- Finished up my demographics onboarding screen and tied together the screens that we have made so far for logging in, onboarding, and displaying match.

## Changes Made


- Completed demographics onboarding screen
- Updated styles for all onboarding screens to match design and some preliminary scaling for different screen sizes
- Combined the flow logic between each screen in onboarding
- Tied together login screen with onboarding, adding some logic to display correct screen when appropriate

## Test Coverage


- Open the app and run through each screen of onboarding.
- Check page navigation for various users (those who haven't signed in yet, those who haven't onboarded yet, those who have done both)

## Next Steps (delete if not applicable)


- We need to make onboarding more dynamic in future iterations once we get actual data from backend. 

## Related PRs or Issues (delete if not applicable)



- #16 Finalize UI of demographics onboarding
- #15 Finalize UI of interests onboarding
- #19 Groups Onboarding Screen
- #10 Preliminary Demographics Screen

## Screenshots (delete if not applicable)



<details>

  <summary>Screen Shot Name</summary>


  
  
![ezgif com-video-to-gif-2](https://user-images.githubusercontent.com/29307883/74906433-0ea70400-537f-11ea-8f6c-0f07cef78d89.gif)


</details>
